documentation/modules/overview/con-metrics-overview-tracing.adoc

```diff
@@ -0,0 +1,18 @@
+// This module is included in:
+//
+// overview/assembly-metrics-overview.adoc
+
+[id="metrics-overview-tracing_{context}"]
+= Distributed tracing
+
+Within a Kafka deployment, distributed tracing using Jaeger is supported for:
+
+* MirrorMaker to trace messages from a source cluster to a target cluster
+* Kafka Connect to trace messages consumed and produced by Kafka Connect
```

Comment:
The tracing is supported for the Kafka Bridge as well.

Reply:
Think I must have been referring to an old guide to miss that.
Added:
_Kafka Bridge to trace messages consumed and produced by Kafka Bridge, and HTTP requests from client applications_